Extensions/xkit_preferences.css

```diff
@@ -1489,3 +1489,19 @@
 	background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABEAAAARCAYAAAA7bUf6AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAyJpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvIiB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIgeG1sbnM6c3RSZWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZVJlZiMiIHhtcDpDcmVhdG9yVG9vbD0iQWRvYmUgUGhvdG9zaG9wIENTNSBNYWNpbnRvc2giIHhtcE1NOkluc3RhbmNlSUQ9InhtcC5paWQ6OTAzMEUyQjUzNzI4MTFFM0E0MTFDNDE2OTFDMzA4QzciIHhtcE1NOkRvY3VtZW50SUQ9InhtcC5kaWQ6OTAzMEUyQjYzNzI4MTFFM0E0MTFDNDE2OTFDMzA4QzciPiA8eG1wTU06RGVyaXZlZEZyb20gc3RSZWY6aW5zdGFuY2VJRD0ieG1wLmlpZDo5MDMwRTJCMzM3MjgxMUUzQTQxMUM0MTY5MUMzMDhDNyIgc3RSZWY6ZG9jdW1lbnRJRD0ieG1wLmRpZDo5MDMwRTJCNDM3MjgxMUUzQTQxMUM0MTY5MUMzMDhDNyIvPiA8L3JkZjpEZXNjcmlwdGlvbj4gPC9yZGY6UkRGPiA8L3g6eG1wbWV0YT4gPD94cGFja2V0IGVuZD0iciI/PuTNYJcAAAE8SURBVHjaYlTrPcGABjSA2AuI7YDYAIhlgPgJEF8A4kNAvA2IbyBrYEEzIAOI04DYEE1cHor9gTgGiGcB8QxshtQCcQMQMzHgByALpgKxKBA3MyBpyCDSABhggqrPgHE0oF4g1gBkg0D6NFiggQgPg5tF5gR1q/edRPaaFxM0FuBg3ZU3DCZTzzBcf/0NRSOIDxIHyaMBOyZoNMJBkI4IQ5W9AkPsqmtwg0A0iA8SB8mjAQMWaDpgQDcIBEAaF4dp4TMABGSYoAmJAd1LlbvugjVef/kNTIP4WLwCAk9YoClRHt2AdjdlDJtB4ljABUZgsi8CMnrJjB0QKGaB5oUYWDSjKSAEzoP0M0EzEygv/CMxsf2D6rsBS6UzoMn4HwkGNMAyIXIGBGWm1zhyMboXcOZimIsOkFqeAAQYAG+iauE8ykdKAAAAAElFTkSuQmCC);
 	border-color: #238ec4;
 }
+
+.xkit--react #xkit_button {
+	display: flex;
+	justify-content: center;
+	margin: 0 16px;
+	position: relative;
```

Comment:
`top: 3px` makes the button visually fall in line with the others but thats the tiniest detail

Reply:
fixed with font-size: 0, below